app/models/phone_number_authentication/authenticate.rb

```diff
@@ -1,5 +1,8 @@
 class PhoneNumberAuthentication
   class Authenticate
+    MAX_FAILED_ATTEMPTS = 5
+    LOCKOUT_TIME = 20.minutes
```

Comment:
Can we make these constants configurable through environment variables?

Reply:
Yup. The overall process for effecting config changes is:
1. Make the updates in the deployment repo and open a PR - [docs](https://github.com/simpledotorg/deployment/tree/master/ansible#changing-secrets)
1. Merge the PR after review (this autudeploys config changes to [five environments](https://semaphoreci.com/resolvetosavelives/deployment)
1. Manually propagate the config changes to other environments - [docs](https://github.com/simpledotorg/deployment/tree/master/ansible#updating-configs)

For steps 1 and 3, You'll need to navigate to the `ansible/` directory of the deployment repo and run the stated commands. The magic "ansible password file" is stored in 1Password. @timcheadle should be able to hook you up with 1Password access.. Look for the "Ansible vault password" entry once you're in. You can stick that anywhere on your computer and use it. Probably not the _best_ way to do that, but it works for now 🤷